src/main/api/engine-api.yaml

```
@@ -388,6 +388,19 @@ paths:
                     schema:
                         type: string
                     example: "my-editorial"
+                -   name: cacheType
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)